tardis/gui/widgets.py

```diff
@@ -1043,14 +1039,7 @@ def get_transition_table(self, lines, atom, ion):
             transitions_count[index] /= float(s)
         for key, value in transitions.items():
             transitions_parsed.append(
-                "%d-%d (%.2f A)"
-                % (
-                    key[0],
-                    key[1],
-                    self.parent.model.atom_data.lines.ix[value[0]][
-                        "wavelength"
-                    ],
-                )
+                f"{key[0]}-{key[1]} {self.parent.model.atom_data.lines.ix[value[0]]["wavelength"]}"
```

Comment:
The appended statement here has been changed. Please check that the statements adhere to their original intended meaning. 
```suggestion
                f"{key[0]}-{key[1]} {self.parent.model.atom_data.lines.ix[value[0]]["wavelength"]:.2f} A"
```